.github/workflows/coverage.yaml

```diff
@@ -2,40 +2,50 @@ name: Code Coverage
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
 env:
   CARGO_INCREMENTAL: 0
   CARGO_TERM_COLOR: always
   RUST_BACKTRACE: 1
 
-defaults:
-  run:
-    shell: bash -ieo pipefail {0}
-
 jobs:
   coverage:
     runs-on: ubuntu-latest
-    container:
-      image: docker://openrr/openrr-coverage-ci:ros2
-      options: --security-opt seccomp=unconfined
-    env:
-      HOME: /root
     steps:
       - uses: actions/checkout@v2
-      - run: ci/ubuntu-install-dependencies.sh
+      - run: |
+          sudo apt-get update
+          sudo apt-get install -y \
+            cmake build-essential libudev-dev xorg-dev libglu1-mesa-dev libasound2-dev libclang-dev \
+            curl gnupg2 lsb-release
+
+          # ros2
+          sudo curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key  -o /usr/share/keyrings/ros-archive-keyring.gpg
+          echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/ros2.list > /dev/null
+          # ros1
+          sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
+          sudo apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
```

Comment:
At first, I don't want to use docker image.
But do you have any reason why not use docker? Just asking.

I mean ros:foxy-ros1-bridge or something, not openrr-ci.

Reply:
I only tried ros:foxy-ros1-bridge and openrr-ci, but after #344, I could not get the build to work properly with the docker.
There seems to be some package installation missing that is needed for FFI, but I could not found out what it is.

Reply:
I see.